com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicMesh.cs

```diff
@@ -302,6 +319,20 @@ public override void AbcSyncDataEnd()
             }
 #endif
 
+            for (var i=0;i<m_splits.Count;++i)
+            {
+                var split = m_splits[i];
+                if (split.active &&  split.velocities.Count > 0)
+                {
+                    var job = new MultiplyByConstant
+                    {
+                        data = split.velocities.Pointer,
+                        scalar = -1
+                    };
+                    m_PostProcessJobs[i] = job.Schedule(split.velocities.Count, 2048);
```

Comment:
Any specific reason you are using a 2048 batch size?

Reply:
empirical constant tweaking.